client/components/term-picker-option/style.scss

```diff
@@ -0,0 +1,77 @@
+.term-picker-option {
+  $padding-size: 18px;
+
+  display: flex;
+  width: 100%;
+  cursor: pointer;
+
+  padding: $padding-size;
+  border-radius: 5px;
+
+  background: #FFF;
+
+  .term-picker-option__radio-wrapper {
+    width: 21px;
+    margin-right: 12px;
+  }
+
+  .term-picker-option__radio {
+    width: 21px;
+    height: 21px;
+
+    &:checked:before {
+      width: 13px;
+      height: 13px;
+    }
+  }
+
+  .term-picker-option__header,
+  .term-picker-option__description {
+    width: 100%;
+  }
+
+  .term-picker-option__term,
+  .term-picker-option__price,
+  .term-picker-option__discount,
+  .term-picker-option__side-note,
+  .term-picker-option__description {
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  .term-picker-option__header {
+    line-height: 24px;
+  }
+
+  .term-picker-option__term {
+    display: inline-block;
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .term-picker-option__discount {
+    margin-left: 10px;
+  }
+
+  .term-picker-option__description {
+    font-size: 16px;
+  }
+
+  .term-picker-option__price {
+    color: #4f646e;
```

Comment:
Please use sass variables found [here](https://github.com/Automattic/wp-calypso/blob/20b63c670ec36b29a593798ad0d1ef5c0a4a531a/assets/stylesheets/shared/_colors.scss) instead of hex values. Hex values aren't allowed.